patterns/2-structured/project-setup/release-process.md

```diff
@@ -0,0 +1,47 @@
+## Title
+
+Standard Release Process/Published Artifacts
+
+## Patlet
+
+Teams may be reluctant to use innersource projects that they are unfamiliar with when there is no clear release process apparent in the repository.
+Providing clear release notes and a published artifact (binary, docker image, jar, etc) gives people confidence you are publishing a quality product.
+
+## Problem
+
+The common code in the shared repository doesn't have a published artifact or publicly viewable release process.
+
+## Context
+
+It is common practice in Open Source projects to have releases, with release notes documenting breaking changes,
+new features, etc along with either a published binary or link to a Docker image. This practice may not be as
+transparent or well documented/visible for inner source projects, modules, etc. Providing robust release notes
+along with a published artifact that is the result of a clearly document and visible release process builds trust and confidence in your project.
+
+## Forces
+
+- Difficult for organizations that don't have a central CI/CD system
+- Adds the burden of publishing release notes
+- No internal location to host artifacts
+
+## Solution
```

Comment:
As you say in the Forces, there is not internal location to host artifacts. Does it make sense to add this here as part of the solution?

- Common / shared place to host all built artifacts

Reply:
Fixed.